it/test/uk/gov/hmrc/incometaxpenaltiesappealsfrontend/controllers/ConfirmationControllerISpec.scala

```diff
@@ -19,9 +19,9 @@ package uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers
 import org.jsoup.Jsoup
 import play.api.http.Status.OK
 import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.stubs.AuthStub
-import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.utils.{ComponentSpecHelper, ViewSpecHelper}
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.utils.{ComponentSpecHelper, NavBarTesterHelper, ViewSpecHelper}
 
-class ConfirmationControllerISpec extends ComponentSpecHelper with ViewSpecHelper with AuthStub {
+class ConfirmationControllerISpec extends ComponentSpecHelper with ViewSpecHelper with AuthStub with NavBarTesterHelper {
 
 
   "GET /appeal-confirmation" should {
```

Comment:
Need to call the `testNavBar` method to add the tests for the NavBar to the ISpec